From f5284e88465cd3ac17129e806ccb784dacca0f93 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:15 -0400
Subject: [redhat] kabi: Adding symbol nla_find (lib/nlattr.c)

Message-id: <be913bb1edfcd3b9dd2efbaaed0dd0a5501a4d5e.1410510773.git.poros@redhat.com>
Patchwork-id: 95965
O-Subject: [PATCH RHEL7.1 135/233] kabi: Adding symbol nla_find (lib/nlattr.c)
Bugzilla: 1077335
RH-Acked-by: Jiri Benc <jbenc@redhat.com>

Bugzilla: 1077335
    https://bugzilla.redhat.com/show_bug.cgi?id=1077335

    Description:
    ============
    Netlink API.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/nla_find
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/nla_find
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/nla_find

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/nla_find  b/redhat/kabi/kabi-module/kabi_ppc64/nla_find 
new file mode 100644
index 0000000..636cd8f
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/nla_find  
@@ -0,0 +1,2 @@
+#1-
+0xcd279169 nla_find vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/nla_find  b/redhat/kabi/kabi-module/kabi_s390x/nla_find 
new file mode 100644
index 0000000..636cd8f
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/nla_find  
@@ -0,0 +1,2 @@
+#1-
+0xcd279169 nla_find vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/nla_find  b/redhat/kabi/kabi-module/kabi_x86_64/nla_find 
new file mode 100644
index 0000000..636cd8f
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/nla_find  
@@ -0,0 +1,2 @@
+#1-
+0xcd279169 nla_find vmlinux EXPORT_SYMBOL
-- 
1.7.1